### DISCUSSION TITLE
rocketmq official website search can't use

### DISCUSSION THREAD

**Discussion by gongxuanzhang** [apache/rocketmq#6896] [General]
https://rocketmq.apache.org/zh/docs/featureBehavior/07messagefilter

I can't use the search function on the official website
<img width="583" alt="image" src="https://github.com/apache/rocketmq/assets/48286053/8430fad9-b87e-46fc-bd8c-1e454ffa5f21">

Please guide me if I have a problem with the way I use it

**Comment by mxsm** [comment]
@RongtongJin @tsunghanjacktsai  是否可以看一下官网algolia搜索是不是没有建立对应的搜索索引
